modules/apps/transfer/types/msgs.go

```diff
@@ -15,6 +15,7 @@ import (
 const (
 	MaximumReceiverLength = 2048  // maximum length of the receiver address in bytes (value chosen arbitrarily)
 	MaximumMemoLength     = 32768 // maximum length of the memo in bytes (value chosen arbitrarily)
+	MaximumTokensLength   = 100   // maximum number of tokens that can be transferred in a single message
```

Comment:
100 seems fine? 👍🏻 could potentially be more, if you had built some kind of workflow doing something on a dex with several different denoms and then sending via transfer or something, but to exceed 100 would also probably be extremely rare, I think its probably safe for now. If someone requests to bump then we can bump!